scripts/predicting_the_past_import.py

```diff
@@ -536,18 +584,20 @@ def write_reactors(in_list, out_path, reactor_template):
                 assem_size = 103000 / assem_no
             config = reactor_template.render(name=name,
                                              lifetime=get_lifetime(row),
+                                             cycletime=cycle_time,
+                                             refueltime=refuel_time,
                                              assem_size=assem_size,
                                              n_assem_core=assem_no,
                                              n_assem_batch=assem_per_batch,
-                                             power_cap=row[3],
+                                             power_cap=row['RUP [MWe]'],
```

Comment:
Sorry, this is a dumb question... 
does this work?

Reply:
yes, because I changed ``row`` to be a DataFrame instead of a list. So this calls the column for the reference unit power (RUP)